### ISSUE TITLE
Add Bidoo

### ISSUE THREAD

**Issue by Pucci00** [jdm-contrib/jdm#318]
I can’t understand how to delete my account in Bidoo.com

**Comment by tupaschoal** [comment]
Please make it clearer what your request is. The way it is phrased, it seems like the instructions we provide aren't clear, but there's no "Bidoo" in our files.
If you want to know how to delete it, we can't really help, generally people find out and tell us so we can populate the page and help others in the future

**Comment by Pucci00** [comment]
I’m sorry. I was asking for instructions. Anyway I find out that the only way to delete an account on the site Bidoo.com is to contact the staff and ask for it.
If you want to add the site to the list I’ll help giving the link of the contact form. Otherwise I’ll just close this issue

**Comment by tupaschoal** [comment]
It's alright. We can either add a link to the contact form or the e-mail, if you have it, but sure, do provide it so we can improve our database. Thanks!

**Comment by Pucci00** [comment]
I can’t get the link because they deleted my account already (without notify me).
Anyway I just opened the contact form asking if they could delete my account.

**Comment by Nightfirecat** [comment]
That's fine, we can work with that information. Do you remember if you needed to give them any specific information for them to delete your account?

**Comment by Pucci00** [comment]
No. I didn’t give any specific information. I just had to confirm with a second message

**Comment by Nightfirecat** [comment]
So the steps that occurred were:

1. You visited the site, opened the contact form, and requested account deletion (without providing any account info?)
2. They messaged you back (how did they do so?)
3. You responded to that message (again, how did that occur--email or something else?)
4. They deleted your account

If you can help fill in the gaps, we can add a card with steps to delete an account from this site. Thanks in advance!

**Comment by Pucci00** [comment]
(1) You can write on the contact form I used (by clicking on the link “assistence” in the website) only after you logged in the website, so I just provided my email address even if unnecessary.

(2) When they reply I got an email which just report that they have replied. I clicked the link on the email, access again on the website and I could see their answer in a sort of a chat page.
There I confirmed my intection to delete the account and nobody replied me or sent me any email, but if I now try to log in with the same email, the account is inexistent.
